src/CoreWCF.WebHttp/src/CoreWCF/Dispatcher/UriTemplateClientFormatter.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Xml;
+using CoreWCF.Channels;
+using CoreWCF.Description;
+
+namespace CoreWCF.Dispatcher
+{
+    internal class UriTemplateClientFormatter : IClientMessageFormatter
+    {
```

Comment:
You should be able to delete this class as we don't need the client formatter.

Reply:
It's being used in WebHttpDispatchOperationSelector.